frontend/package.json

```diff
@@ -14,12 +14,12 @@
     "format": "prettier --write src/"
   },
   "dependencies": {
+    "axios": "^1.6.8",
```

Comment:
Please avoid adding new dependencies.
In this case you can create a put function in http.ts and call it thorugh the actions.ts.
Look at the HomeView.ts rolesStore.fetchRoles() example.